guardian/ophan-geoip-db-refresher
# GuCDK GeoIP DB Refresher

## What does this change?

This PR migrates Ophan GeoIP DB Refresher to GuCDK which is tracked in [#5086](https://github.com/guardian/ophan/issues/5086). It is a scheduled Lambda with cron job so we used `GuScheduledLambda`.  

## How to test

We will update the CloudFormation in AWS then we will do a manual deployment in riff-raff with this branch and our final step is to trigger the Lambda manually and check ELK logs. 

## How can we measure success?

We have the Ophan GeoIP DB Refresher using GuCDK!

## Have we considered potential risks?

There is no high risk for this PR and the worst case scenario is the file with MaxMind data in our S3 bucket won't be updated but that should be noticed during our testing phase when we trigger the Lambda manually and we will keep an eye on the cron job which should be running on the following day (Wed 1st of March) of our migration.
